xwalk_android.gypi

```diff
@@ -213,6 +213,59 @@
       'includes': ['../build/jni_generator.gypi'],
     },
     {
+      'target_name': 'xwalk_runtime_lib_lzma_compress',
+      'type': 'none',
+      'dependencies': [
+        'xwalk_runtime_lib_apk',
+      ],
+      'variables': {
+        'files_to_compress': [
+          '<(PRODUCT_DIR)/xwalk_runtime_lib_apk/stripped_libraries/libxwalkcore.so',
+          '<(PRODUCT_DIR)/xwalk_runtime_lib_apk/classes.dex',
+          '<(PRODUCT_DIR)/xwalk_runtime_lib/assets/icudtl.dat',
+          '<(PRODUCT_DIR)/xwalk_runtime_lib/assets/xwalk.pak',
+        ],
+        'assets_dir': '<(PRODUCT_DIR)/xwalk_runtime_lib_lzma/assets/',
+      },
+      'actions': [
+        {
+          'action_name': 'lzma_compress',
+          'message': 'compress ingredients for runtime APK',
+          'inputs': [
+            'build/android/lzma_compress.py',
+          ],
+          'outputs': [
```

Comment:
I am a little bit confused by the mode mentioned here. Could you please elaborate? Those output file name will be automatically with ".lzma" appended when running the python script ```lzma_compress.py```. The action outputs is needed only for setup the ninja build dependency, please see below.

```
 build xwalk_runtime_lib_lzma/gen.stamp $
    xwalk_runtime_lib_lzma/assets//libxwalkcore.so.lzma $
    xwalk_runtime_lib_lzma/assets//classes.dex.lzma $
    xwalk_runtime_lib_lzma/assets//icudtl.dat.lzma $
    xwalk_runtime_lib_lzma/assets//xwalk.apk.lzma: $
xwalk_runtime_lib_lzma_compress_target_lzma_compress_6e0c00b19991794c1889fe40f5e2db2d $
    ../../xwalk/build/android/lzma_compress.py || $
    obj/xwalk/xwalk_runtime_lib_apk.actions_rules_copies.stamp
```